kubeapiversion/api_version_test.go

```diff
@@ -0,0 +1,134 @@
+package kubeapiversion_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	"github.com/solo-io/go-utils/kubeapiversion"
+)
+
+var _ = Describe("ApiVersion", func() {
+	Describe("ParseApiVersion", func() {
```

Comment:
Could you possibly also test some other error versions so it's a little more clear what versions are bad, also, maybe a link to kube documentation about how these versions work, and how our library differs from their sorting